.changeset/serious-meals-wink.md

```diff
@@ -0,0 +1,9 @@
+---
+"@clerk/shared": patch
```

Comment:
🔧 I think that changing the public API (eg removing an export in our case) of a package requires a major version bump.

Reply:
It's an internal package without a README only used by our Jest tests that we also don't tell people to install or use. It'd consider it purely internal.

So while technically you're correct, I wouldn't care about that :)